### DISCUSSION TITLE
General context for simssadb

### DISCUSSION THREAD

**Discussion by malajvan** [DDMAL/linkedmusic-datalake#13] [Show and tell]
Since we have a number of properties which overlaps between multiple databases, it is beneficial to have a running list of them for other incoming database and make the process more standardized.
You can find the files [here](https://github.com/malajvan/linkedmusic-datalake/tree/main/general)
Currently we have:
[P1476](http://www.wikidata.org/prop/direct/P1476): title
[P86](http://www.wikidata.org/prop/direct/P86): composer
[P136](http://www.wikidata.org/prop/direct/P136): genre
[P826](http://www.wikidata.org/prop/direct/P826): tonality (key)
[database](https://schema.org/Dataset): schema.org Dataset
[P2701](http://www.wikidata.org/prop/direct/P2701): file format
[P577](http://www.wikidata.org/prop/direct/P577): publication date
[P2699](http://www.wikidata.org/prop/direct/P2699): URL
[P50](http://www.wikidata.org/prop/direct/P50): author
[P175](http://www.wikidata.org/prop/direct/P175): performer
[P2550](http://www.wikidata.org/prop/direct/P2550): recording or performance of
[P1545](http://www.wikidata.org/prop/direct/P1545): series ordinal

I would appreciate it if others could give me feedback as well as point out any missing or inappropriate properties.

**Comment by fujinaga** [comment]
This is a great list. At this point we are not trying to be exhaustive, so let's stick with these for now.
